### ISSUE TITLE
Arming does not work - lowPrivilege

### ISSUE THREAD

**Issue by HappyPlotter** [petrleocompel/hikaxpro_hacs#107]
Hello, just installed the addon (thank you for this). It seems like the arming is not working. The error is :

```
Logger: homeassistant.helpers.script.websocket_api_script
Source: helpers/script.py:507
First occurred: 11:02:55 PM (3 occurrences)
Last logged: 11:04:22 PM

websocket_api script: Error executing script. Unexpected error for call_service at pos 1: Unexpected response status code 400 returned with message { "statusCode": 4, "statusString": "Invalid Operation", "subStatusCode": "lowPrivilege", "errorCode": 1073741826, "errorMsg": "lowPrivilege" }
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 507, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 736, in _async_call_service_step
    response_data = await self._async_run_long_action(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 699, in _async_run_long_action
    return await long_task
           ^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2543, in async_call
    response_data = await coro
                    ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2580, in _execute_service
    return await target(service_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 996, in entity_service_call
    raise result from None
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1580, in async_request_call
    return await coro
           ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 1043, in _handle_entity_call
    result = await task
             ^^^^^^^^^^
  File "/config/custom_components/hikvision_axpro/alarm_control_panel.py", line 236, in async_alarm_arm_away
    await self.coordinator.async_arm_away(self.sys.id)
  File "/config/custom_components/hikvision_axpro/__init__.py", line 319, in async_arm_away
    is_success = await self.hass.async_add_executor_job(self.axpro.arm_away, sub_id)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/hikaxpro.py", line 197, in arm_away
    return self._base_json_request(f"http://{self.host}{consts.Endpoints.Alarm_ArmAway.replace('{}', sid)}",
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/hikaxpro.py", line 180, in _base_json_request
    raise errors.UnexpectedResponseCodeError(response.status_code, response.text)
errors.errors.UnexpectedResponseCodeError: Unexpected response status code 400 returned with message { "statusCode": 4, "statusString": "Invalid Operation", "subStatusCode": "lowPrivilege", "errorCode": 1073741826, "errorMsg": "lowPrivilege" }
```

The user is the admin and the "include areas as separate zones" is checked

**Comment by petrleocompel** [comment]
I guess you are using your Hikvision accounts credentials. What version of firmware do you have ? what device exactly it is ?

**Comment by HappyPlotter** [comment]
That is correct, the account is from the cloud.
The device is DS-PWA96-M2H-WE
Device Version V1.2.9 build 231207

Unfortunately it does not allow to create a local account, it wont allow to enter the password

**Comment by petrleocompel** [comment]
Can you login in Web UI of AXPro with this account ?

**Comment by HappyPlotter** [comment]
Yes, there is no issues connecting to the web ui with this account. Also all the devices are being seen by HA. It is just the arming that has issues (probably disarming too)

**Comment by petrleocompel** [comment]
Can you take a screenshot of users page? (just hide your email and name from that)

**Comment by HappyPlotter** [comment]
![image](https://github.com/petrleocompel/hikaxpro_hacs/assets/111639418/6afb7283-cb55-47ee-a4fc-bcc6bb9ddab1)

**Comment by petrleocompel** [comment]
So you are logging in with user 501.

**Comment by HappyPlotter** [comment]
Correct

**Comment by petrleocompel** [comment]
Can you post screenshot of the configuration from HA ? (again anonymise your details)
![image](https://github.com/petrleocompel/hikaxpro_hacs/assets/9423543/4ed666cd-a6b6-4872-b89b-2e19c3121567)

**Comment by HappyPlotter** [comment]
![image](https://github.com/petrleocompel/hikaxpro_hacs/assets/111639418/81e06a75-3939-4fd8-a7a1-bf091aec7f35)

**Comment by petrleocompel** [comment]
Can you try to disable option for including areas ? and then try to arm ? 
And which "zone" are you trying to arm ?

**Comment by petrleocompel** [comment]
But option "use code for arming" is disabled. So please do not input the code.

**Comment by HappyPlotter** [comment]
The AX pro as a whole. I still get the error

![image](https://github.com/petrleocompel/hikaxpro_hacs/assets/111639418/05d13bfb-5d92-4352-b204-f516c69f3fcc)

**Comment by HappyPlotter** [comment]
I have tried all the possible ways, with code, without code, with zones and without zones

**Comment by petrleocompel** [comment]
I will have to check my firmware version and get back to this to review what is happening.

**Comment by petrleocompel** [comment]
> I have tried all the possible ways, with code, without code, with zones and without zones

I guess results were all time the same "lowPrivilege".

**Comment by HappyPlotter** [comment]
There is also one other warning that I see 

This error originated from a custom integration.

Logger: custom_components.hikvision_axpro.model
Source: custom_components/hikvision_axpro/model.py:780
integration: hikvision_axpro
First occurred: 8:45:18 PM (40 occurrences)
Last logged: 9:41:30 PM

Invalid detector_wiring_mode noEOL

**Comment by petrleocompel** [comment]
Which version of integration you are using i guess `2.1.1`?

**Comment by HappyPlotter** [comment]
2.1.1, it is a fresh (3 week) installation

**Comment by HappyPlotter** [comment]
This issue is related to #108 after removing the "installer" from the user, it is arming and disarming

**Comment by petrleocompel** [comment]
Closed via #108
